_posts/2017-10-05-Kickstarter-Trajectory-2017.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+authors: [nick_geudens,ken_de_becker,michiel_willems,jef_willems,simon_buysse]
+title: "Kickstarter Trajectory 2017"
+image: /img/kicks.png
+tags: [Spring, Angular2, Unit Test, Microservices, Spring Boot, Ionic, MongoDB]
+category: Kickstarters
+comments: true
+---
+
+>This year, five talented people started the Ordina Kickstarter trajectory.
+Each of them looking for a challenge and a fresh start!
+For some, it was a transition between school and work, and Ordina handled this very well.
+The main goal of this trajectory was to widen every student’s knowledge of the IT world.
+They taught us the basics of every topic that’s hot at the moment.
+This will definitely come in handy during our first project.
+
+
+# First impressions
```

Comment:
Please have a look at this paragraph, the first sentence and last sentence are fine. The other 4 have the same structure, which feels repetitive.